docs/building-extensions/language-files/index.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 9
+title: Language Files
+---
+
+This section describes the use of language files for extensions. Joomla! language definition files are written in the
+very basic [INI file format](https://en.wikipedia.org/wiki/INI_file). They must be saved using the **UTF-8 encoding**.
+Blank lines and lines beginning with a
+semicolon (;) are ignored and the latter may be used to add comments to the file. Each line consists of a key-value pair
+separated by an equals sign:
+
+```ini
+KEY="value"
+```
+
+where `KEY` is a string to be translated and `value` is the translated string. For example:
+
+```ini
+COM_EXAMPLE_ADDITIONAL_INFO_LABEL="Additional Information"
+```
+
+## Conventions & Specifications for Language Keys
+The `KEY` should only use **ASCII characters**. It should only contain capital letters, digits, underscores and hyphens,
```

Comment:
I think core doesn't use hypens nor dots. I think shouldn't be a suggestion but also don't hurt.